docs/pages/kubernetes-access/manage-access/rbac.mdx

```diff
@@ -0,0 +1,452 @@
+
+For valid requests, the Kubernetes Service rewrites the request headers to
+impersonate the Teleport user's desired Kubernetes user and groups, and forwards
+the request to the API server.
+
+In this section, we will define a Teleport role that:
+
+- Authenticates the user to a Kubernetes cluster as a member of the `developers`
+  group. In the previous section, we authorized members of this group to view
+  pods in all namespaces. 
+- Enables the user to access `webapp` pods in the `production` namespace and all
+  pods in the `development` namespace.
+- Denies the user access to all other pods.
+
+### Define a role
+
+Create a file called `kube-access.yaml` with the following content:
+
+```yaml
+kind: role
+metadata:
+  name: kube-access
+version: v6
+spec:
+  allow:
+    kubernetes_labels:
+      'region': '*'
+      'platform': 'minikube'
+    kubernetes_resources:
+      - kind: pod
+        namespace: "production"
+        name: "^webapp-[a-z0-9-]+$"
+      - kind: pod
+        namespace: "development"
+        name: "*"
+    kubernetes_groups:
+    - developers
+    kubernetes_users:
+    - minikube
+  deny: {}
+```
+
+In this role, we have defined the following `allow` rules:
+
+- `kubernetes_labels`: Allows access to Kubernetes clusters in all regions, but
+  only with the `platform:minikube` label.
+- `kubernetes_resources`: Allows access to pods in the `webapp` deployment in
+  the `production` namespace and all pods in the `development` namespace. Note
+  the use of a regular expression (beginning `^` and ending in `$`) to match pod
+  names that Kubernetes generates automatically.
+- `kubernetes_groups`: Authenticates the user to your Kubernetes cluster as a
+  member of the Kubernetes group `developers`, which we associated with the
+  `pod-viewer` Kubernetes `Role` earlier in this guide.
+- `kubernetes_users`: Authenticates the user to your Kubernetes cluster as the
+  default `minikube` user.
+  
+### Create the role
+
+Once you have finished configuring the `kube-access` role, create it using the
+following command:
+
+```code
+$ tctl create kube-access.yaml
+```
+
+(!docs/pages/includes/add-role-to-user.mdx role="kube-access"!)
+
+## Step 3/3. Access resources
+
+At this point, you have configured the Teleport Kubernetes Service to give your
+Teleport user access to the `webapp` pod in the `production` namespace. In this
+step, we will authenticate to your Kubernetes cluster via Teleport and test our
+new access controls.
+
+List the Kubernetes clusters you can access via Teleport:
+
+```code
+$ tsh kube ls
+```
+
+You should see the `minikube` cluster you registered earlier:
+
+```text
+Kube Cluster Name Labels                         Selected
+----------------- ------------------------------ --------
+minikube          platform=minikube region=local
+```
+
+To access your Kubernetes cluster via Teleport, authenticate to it and update
+your kubeconfig:
+
+```code
+$ tsh kube login minikube
```

Comment:
As above, should this be `kubectl`?

Reply:
I've verified that this command is correct